src/lib/docs/Templates/LoginPlugin.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Login Plugin Template
+description: A foundational template designed for the creation of login plugins integrated with the Wharf SessionKit.
+category: Templates
+published: true
+order: 3
+---
+
+# Login Plugin Template
+
+The Login Plugin Template is designed to provide developers with a foundational structure to craft their own [login plugins](/docs/session-kit/plugin-login) for the [Wharf SessionKit](/docs/session-kit).
+
+## Usage
+
+### Cloning the Repository
+
+To use this template:
```

Comment:
Maybe on each of these, we can include two options:

1. Use the Github template clone feature
2. Use what you wrote here

It's a lot easier to just click a button and create a new repo on Github:

<img width="1402" alt="image" src="https://github.com/wharfkit/website/assets/677686/a5544f4c-b96f-4dba-9f6d-e75f0fc9fa36">


https://github.com/wharfkit/wallet-plugin-template